dctoolbox/slam_interface_convertor.py

```diff
@@ -224,7 +224,8 @@ class InterfaceAdaptorConfig:
     slam_json_path: Path
 
     output_path: Path
-
+    
+    num_cameras: int = None
```

Comment:
doing this: num_cameras: int = field(init=False)

Reply:
done